assets/i18n/en_US.json

```
@@ -49,7 +49,7 @@
         "widgetTitle": "Patcher",
         "patchButton": "Patch",
         "patchDialogText": "You have selected a resource patch and a split APK installation has been detected, so patching errors may occur.\nAre you sure you want to proceed?",
-        "armv7WarningDialogText": "Patching on ARMv7 devices is not yet supported. Proceed anyways?"
+        "armv7WarningDialogText": "Patching on ARMv7 devices is not yet supported. Patching will likely fail or abort. Proceed anyways?"
```

This is worded better:

```suggestion
        "armv7WarningDialogText": "Patching on ARMv7 devices is not yet supported and might fail. Proceed anyways?"
```